scripts/getting-started.sh

```diff
@@ -114,8 +114,16 @@ if command -v rustc >/dev/null 2>&1; then
     echo "\n✅︎🦀 Rust already installed."
 else
     if prompt_default_yes "\n🦀 Rust is not installed. Install it?"; then
-        echo "🦀 Installing via rustup."
-        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+        version=`cat ../.github/env | grep IMAGE | cut -d'-' -f3`
```

Comment:
But also, this won't if you are not in the polkadot sdk repo right? this `.github/env` is not available otherwise. 

In other words, have you tried the outcome, and does it actually works?

Reply:
I would argue that 99% of the users running this script are outside of the repo (recall the script is meant to onboard a new user), so I think the solution presented here is really marginally better than not having it.

I think that having a rust-toolchain file is probably the better approach. In that case all devs can inspect this file and adjust their rust versions accordingly. Hopefully IDEs can even automatically warn you about it.

Reply:
True, and it _could_ create complications if my proposed script (God forbid) finds "1.2.0" and installs + uses that version. 

My worry is that since this is a getting-started script, users might not know Rust inside and out (the script installs Homebrew & Rust, amongst others) and therefore not even know to add it to their project at all. It might be useful to add the following line to fetch the toml file directly from polkadot-sdk, once it's added in:

`curl -s -H "Accept:application/vnd.github.v3.raw" https://api.github.com/repos/paritytech/polkadot-sdk/contents/rust-toolchain.toml > rust-toolchain.toml`

I pushed an example of this.

Reply:
I'm forgetting that this script is used to create the environment, so bringing in the file is not so useful. Perhaps pulling directly from this script would be better.

Reply:
@kianenigma - After some guidance from Nazar, a decision was made to pull the `rust-toolchain.toml` from polkadot-sdk straight into the minimal-template fetched at the end of the script, which should satisfy the requirement. This PR now contains a base toml file with the required version to support this.